packages/flutter/test/material/text_field_test.dart

```diff
@@ -1614,19 +1607,75 @@ void main() {
       );
     }
 
-    // Explicitly allow selection on obscured text
-    await tester.pumpWidget(buildFrame(true, true));
+    // Explicitly disabled selection on obscured text.
+    await tester.pumpWidget(buildFrame(true, false));
     await tester.enterText(find.byType(TextField), 'abcdefghi');
     await skipPastScrollingAnimation(tester);
     expect(controller.selection.isCollapsed, true);
 
-    // Long press does select text
+    // Long press does select text.
     final Offset ePos2 = textOffsetToPosition(tester, 1);
-    final TestGesture gesture2 = await tester.startGesture(ePos2, pointer: 7);
-    await tester.pump(const Duration(seconds: 2));
-    await gesture2.up();
+    await tester.longPressAt(ePos2, pointer: 7);
     await tester.pump();
-    expect(controller.selection.isCollapsed, false);
+    expect(controller.selection.isCollapsed, true);
```

Comment:
Can you explain this section here?  Doesn't "isCollapsed true" mean that the text is not selected, or am I reading this wrong?

Reply:
if it is selectable, a long press will cause it to select the whole word and the selection will not be collapsed.

I inverted this test and the test above because we are inverted the selection behavior of password textfield.

Reply:
Are the comments reversed then?  Here it says "Long press does select text." and then it asserts that isCollapsed is true, meaning that there is no selection.  Above it says "Long press doesn't select anything." and then asserts isCollapsed is false.  Or maybe I'm just confusing myself haha.

Reply:
oh I forgot to update the comment, thanks for catching it